package.json

```diff
@@ -4,19 +4,20 @@
   "description": "Complete, flexible, extensible and easy to use page transition library for your web.",
   "main": "lib/index.js",
   "scripts": {
-    "cy:open": "cypress open",
     "compile": "babel --presets es2015,stage-0 -d lib/ src/",
     "build": "webpack-cli",
     "lint": "prettier src/**/*.{js,mjs} --write",
     "prepublish": "npm run compile && npm run build",
     "postinstall": "opencollective-postinstall || true",
-    "ci": "npm run build && npm run test:instrument && start-server-and-test test:server http://localhost:8274 test:run:record",
-    "test": "npm run build && npm run test:instrument && start-server-and-test test:server http://localhost:8274 test:run",
-    "test:headed": "npm run build && npm run test:instrument && start-server-and-test test:server http://localhost:8274 cy:open",
+    "ci": "start-server-and-test test:start 8274 cy:run:record",
+    "test": "start-server-and-test test:start 8274 cy:run",
+    "test:dev": "start-server-and-test test:start 8274 cy:open",
```

Comment:
Am I missing something, or are the port numbers duplicated and don't actually do anything here?

Reply:
They are duplicated but I can't think of a good way of sharing them. They're also hardcoded into the test suite to check the domain/origin, we're hardcoding them here as well to make sure it's always the same port on each run.